src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js

```diff
@@ -42,11 +49,12 @@ export default class geFormWidgetTokenizeCard extends LightningElement {
     _currentPaymentMethod = undefined;
     _hasPaymentMethodInTemplate = false;
 
-    @api
+
```

Comment:
I believe it's best practice to add the decorator to the getter. Any particular reason we're moving it to the setter?

Reply:
I think I should remove the `get` keyword from `widgetDataFromState` since it's only used in the js file, not even in the markup. The setter is how the sliced `widgetState` is passed down from `geFormWidget`